deps/Makefile

```diff
@@ -485,12 +493,24 @@ LLVM_MFLAGS += OPTIONAL_PARALLEL_DIRS=clang
 else
 # block default building of Clang
 LLVM_MFLAGS += OPTIONAL_PARALLEL_DIRS=
+ifeq ($(LLVM_VER),svn)	# changed since LLVM 3.9
```

Comment:
Version checks in Makefiles are reasonably hard to do properly and in a portable manner...